juniper-vpn.py

```diff
@@ -234,7 +237,8 @@ def action_key(self):
                 sys.exit(1)
             self.key = hotp(self.args.oath)
         elif self.key is None:
-            self.key = getpass.getpass('Two-factor key:')
+            matrix = self.get_matrix_index(self.br.response().read())
+            self.key = getpass.getpass('Two-factor key (' + matrix + '):')
```

Comment:
Is there a way to determine if a standard time based two factor key is required or a matrix card one and prompt accordingly?